devicon.json

```
@@ -5706,6 +5706,26 @@
         "color": "#212177",
         "aliases": []
     },
+    {
+        "name": "pfsense",
+        "altnames": ["pfSense"],
+        "tags": ["network"],
+        "versions": {
+            "svg": ["original", "original-wordmark"],
+            "font": ["original", "original-wordmark"]
+        },
+        "color": "#000000",
+        "aliases": [
+            {
+            "base": "original",
+            "alias": "plain"
+            },
+            {
+            "base": "original-wordmark",
+            "alias": "plain-wordmark"
+            }
+        ]
+    },
```

Restructure and add some tags.
```suggestion
        "altnames": [
            "pfSense"
        ],
        "tags": [
            "cloud",
            "network",
            "open-source",
            "security",
            "software"
        ],
        "versions": {
            "svg": [
                "original",
                "original-wordmark"
            ],
            "font": [
                "original",
                "original-wordmark"
            ]
        },
        "color": "#000000",
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            },
            {
                "base": "original-wordmark",
                "alias": "plain-wordmark"
            }
        ]
    },
```